lib/parser.js

```diff
@@ -66,6 +66,15 @@ const args = [
           'on this server'
   }],
 
+  [['-ct', '--connection-timeout'], {
+    required: false,
+    default: null,
+    dest: 'keepAliveTimeout',
+    type: 'int',
+    help: 'Number of seconds the Appium server should apply as the connection timeout for all ' +
```

Comment:
We should mention the default value

Reply:
connection timeout is something different. Keepalive works on the lower level than HTTP does (TCP one)

Reply:
we use the same value for both, in the code, though...?